frontend/src/components/comments/CommentCard.jsx

```diff
@@ -8,101 +8,121 @@ import CommentUpdateForm from "./CommentUpdateForm";
-    return commentCreatorEmail === currentUser.email;
-  }
 
-  function checkForCommentLikeUser() {
-    const likeUserEmail = listOfCommentLikedUsers.find(
-      (user) => user.email === currentUser.email
-    );
-    return likeUserEmail != null;
-  }
+    function checkCommentUserEmail() {
+        return commentCreatorEmail === currentUser.email;
+    }
 
-  return (
-    <div>
-      <span>{commentCreatorName}: </span>
-      <span className="word-wrap">{comment.contentText} </span>
-      <span>
-        <Moment fromNow>{comment.createdTime}</Moment>
-      </span>
-      <p>{comment.listOfCommentLikes.length} comment like(s)</p>
-      <button onClick={commentLikeAction}>
-        {checkForCommentLikeUser() ? "Remove Comment Like" : "Like Comment"}
-      </button>
+    function checkForCommentLikeUser() {
+        const likeUserEmail = listOfCommentLikedUsers.find(
+            (user) => user.email === currentUser.email
+        );
+        return likeUserEmail != null;
+    }
 
-          {checkCommentUserEmail() && (
-              <div>
-                  <button onClick={handleDelete}>Delete</button>
-                  <button
-                      onClick={() =>
-                          toggle ? setToggle(false) : setToggle(true)
-                      }
-                  >
-                      {toggle ? "Cancel Update" : "Update"}
-                  </button>
-                  {toggle && (
-                      <CommentUpdateForm
-                          onSubmit={(commentData) => updateComment(commentData)}
-                          comment={comment}
-                      />
-                  )}
-              </div>
-          )}
-      </div>
-  );
+    return (
+        <div className="comment-card">
+            <div className="postcard-header">
+                <div className="post-info">{commentCreatorName}</div>
+                <div className="delete-edit-icons">
+                    {checkCommentUserEmail() && (
+                        <span>
+                            <button
+                                className="button-post-card delete-icon"
+                                onClick={handleDelete}
+                            />
+                            <button
+                                className={`button-post-card ${
+                                    toggle ? " cancel" : " active"
+                                }`}
+                                onClick={() =>
+                                    toggle ? setToggle(false) : setToggle(true)
+                                }
+                            />
+                        </span>
+                    )}
+                </div>
+                <Moment className="time-lapse" fromNow>
+                    {comment.createdTime}
+                </Moment>
+            </div>
+            <div className="word-wrap comment-content">
+                {!toggle ? (
+                    comment.contentText
```

Comment:
I like what you did with the toggling the view of the contentText or update form!

Perhaps we could wrap `comment.contentText` in `<p>` tags so it inherits the same font family as post contentText?

Reply:
Done 
![Screenshot 2021-05-11 at 13 40 34](https://user-images.githubusercontent.com/43238452/117809467-813f2080-b25e-11eb-95e9-cdd37773f331.png)